chapter_appendix_tools/aws.md

```diff
@@ -231,70 +231,57 @@ First, to simplify the installation, you need to install [Miniconda](https://con
 
 ```bash
 wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
-sudo sh Miniconda3-latest-Linux-x86_64.sh
+sudo sh Miniconda3-latest-Linux-x86_64.sh -b
 ```
 
 
-You need to answer the following questions:
+Next initialize the shell so we can run `conda` direclty.
 
 ```bash
-Do you accept the license terms? [yes|no]
-[no] >>> yes
-
-Miniconda3 will now be installed into this location:
-/home/ubuntu/miniconda3
-  - Press ENTER to confirm the location
-  - Press CTRL-C to abort the installation
-  - Or specify a different location below
->>> <ENTER>
-
-Do you wish the installer to initialize Miniconda3
-by running conda init? [yes|no]
-[no] >>> yes
+~/miniconda3/bin/conda init
 ```
 
 
-After miniconda installation, run the following command to activate CUDA and Conda.
+Next, download the code for this book.
 
 ```bash
-source ~/.bashrc
+sudo apt-get install unzip
 ```
 
 
-Next, download the code for this book.
-
 ```bash
-sudo apt-get install unzip
 mkdir d2l-en && cd d2l-en
-wget http://numpy.d2l.ai/d2l-en.zip
+```
+
+
+```bash
+wget https://d2l.ai/d2l-en.zip
 unzip d2l-en.zip && rm d2l-en.zip
 ```
 
 
-Then create the conda `d2l` environment and enter `y` for the proceed inquiry as shown in :numref:`fig_conda_create_d2l`.
+Then close and re-open your current shell, you should be able to create a new
+environment as following now:
 
 ```bash
-conda create --name d2l
+conda create --name d2l -y
 ```
 
 
-![ Conda create environment D2L. ](../img/conda_create_d2l.png)
-:width:`700px`
-:label:`fig_conda_create_d2l`
 
-After create `d2l` environment, activate it and install `pip`.
+After create `d2l` environment, activate it and install `pip` and `python`. Note that currently we need to install python3.7, since python3.8 is still at its testing period and is not  compatible with some packages.
```

Comment:
remove: Note that currently we need to install python3.7, since python3.8 is still at its testing period and is not  compatible with some packages.